yt_dlp/extractor/common.py

```diff
@@ -2712,11 +2712,17 @@ def extract_Initialization(source):
                     mime_type = representation_attrib['mimeType']
                     content_type = representation_attrib.get('contentType', mime_type.split('/')[0])
 
-                    codecs = representation_attrib.get('codecs', '')
+                    codecs = parse_codecs(representation_attrib.get('codecs', ''))
                     if content_type not in ('video', 'audio', 'text'):
                         if mime_type == 'image/jpeg':
                             content_type = mime_type
-                        elif codecs.split('.')[0] == 'stpp':
+                        # XXX: is this okay for mixed streams (video+audio, audio+subtitle)?
+                        # is it even possible to ever encounter them?
```

Comment:
this is fine. The audio/video content_type is only used for creating format_id. If there is video+sub (or audio+sub), it makes sense to label it as video/audio and not as sub